adapters/bidder_test.go

```diff
@@ -52,12 +52,12 @@ type mockConversions struct {
 	mock.Mock
 }
 
-func (m mockConversions) GetRate(from string, to string) (float64, error) {
+func (m *mockConversions) GetRate(from string, to string) (float64, error) {
```

Comment:
I'm fine with this change to make `go vet` happy, but the existing code is perfectly fine since it's just a mock and doesn't utilize the lock.